dashboard/db/schema.rb

```diff
@@ -39,6 +39,24 @@
     t.index ["lesson_activity_id"], name: "index_activity_sections_on_lesson_activity_id"
   end
 
+  create_table "ai_tutor_interactions", charset: "utf8", collation: "utf8_unicode_ci", force: :cascade do |t|
+    t.integer "user_id", null: false
```

Comment:
IIRC we want a primary key/id column on this table. Do we need to add it here explicitly?

Reply:
An auto-incrementing `id` column comes from Rails magic 🪄 `create_table` will implicitly create a primary key called "id".

<img width="792" alt="Screenshot 2024-01-24 at 11 33 43 AM" src="https://github.com/code-dot-org/code-dot-org/assets/12300669/96a2659c-cfbb-4923-9fe8-d3cf748223ce">